tailwind.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     "./storybook/components/**/*.tsx",
   ],
   presets: [Preset],
+  important: "body",
```

Comment:
I realized that classNameOerride with Tailwind utility classes did not have specificity and was no applying in the story without `!`'s if the style was previously declared. I think this is just in the way we consume tailwind in our repo not a reflection of specificity problems in how we ship our components, which this config won't impact

Reply:
Yeah our Storybook TW is flawed 😬 That's a different task to tackle though haha.